### DISCUSSION TITLE
Am i doing it right?

### DISCUSSION THREAD

**Discussion by Tonqe** [swar/Swar-Chia-Plot-Manager#128] [Q&A]
Hello, pretty new to the chia world and just set up my pc to mine chia, i tried this config but i'm sure i'm not doing everything allright and not pushing as much as i can, i would love some expert to help me out here.. my specs are: i5 10700 (8c/16t), 32gb ram, 1x wd black an1500 and 1x samsung 970evo as nvme devices and this is my actual config:

    name: WD_BLACK
    size: 32
    bitfield: true
    threads: 3
    buckets: 128
    memory_buffer: 2500
    max_concurrent: 6
    max_concurrent_with_start_early: 7
    stagger_minutes: 100
    max_for_phase_1: 3
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false
    
    name: 970EVO
    size: 32
    bitfield: true
    threads: 3
    buckets: 128
    memory_buffer: 2500
    max_concurrent: 6
    max_concurrent_with_start_early: 7
    stagger_minutes: 100
    max_for_phase_1: 3
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false

**Comment by DerJungeHerr** [comment]
Max Phase 1 could be to high.... but on the other hand you have a delay of 3h to start next plot 🤔 I'm also playing with different options since few days ... I guess it's. normal until you have the right parameters.

**Reply by Tonqe** [reply]
changed a bit the config, updated it in the first post, i had to limit the concurrent plots cause i didnt calculate the total ram needed well, guess i will take 32gb more.. For the delay ye, was absolutely too much, it takes like 3h 20min to finish phase 1, so i putted 100min now so if i have understood well, each nvme will start two phase 1 max every time. But i still think i can do way better with a better config
